### ISSUE TITLE
WooCommerce duplicate product compatiblity

### ISSUE THREAD

**Issue by aaronware** [10up/distributor#737]
**Describe the bug**
When utilizing the built-in duplicate product functionality of WooCommerce, If you are on a connected child site and duplicate the product, it will keep the distributor meta, so you will have 2 products that are now connected to the original product from the origin site.

**Expected behavior**
Duplicated product on child site should not have a connection to the origin distribution.

**Environment information**
 - WordPress version: 5.7
 - WooCommerce 5.1
 - Distributor  1.6.3-dev

**Additional context**
Not sure if this should be a compatibility implemented on the WooCommerce side or on the Distributor side.

**Comment by jeffpaul** [comment]
@aaronware could you explain your use case a little more where you'd want one product in sync but a duplicate of it not in sync on a remote site?  Better understanding your scenario will help us better think through a potential resolution should that be feasible and warranted on the Distributor side... thanks!

**Comment by jmslbam** [comment]
I think what he means, is that the Woocommerce duplicate functionality also copies the distributor meta data, so now 2 posts have the same connection mapping.

I'm using https://yoast.com/wordpress/plugins/duplicate-post/ plugin on a site, and there you can add custom meta fields keys to ignore when duplicating a post. I think that @aaronware best can check if the built-in Woo functionality has a similar functionality to ignore certain field keys.

The question is, do you as, Distributor want to add this to Core so all future User can benefit from it, or simply document the fix for now :)

I'm writing a little bit of code now for the Yoast Plugin, will share here when finished.

**Comment by jmslbam** [comment]
Here is my solution https://github.com/burovoordeboeg/distributor-meta-data/commit/5cc2e966b5934f82aae789f28370989798498573

**Comment by jeffpaul** [comment]
@helen do you have a preference on the resolution route to take on this?

**Comment by jeffpaul** [comment]
@jmslbam thanks for your careful thought and code on this, its greatly appreciated!

**Comment by aaronware** [comment]
@jmslbam yes thank you so much for clarifying. I put in the issue and meant to go back and clarify a bit but was completely pulled away on something else.
